backend/core/views.py

```
@@ -2178,6 +2186,26 @@ def destroy(self, request, *args, **kwargs):
 
         return super().destroy(request, *args, **kwargs)
 
+    @action(
+        detail=True, name="Get user tasks", url_path="tasks/(?P<start>.+)/(?P<end>.+)"
+    )
+    def tasks(
+        self,
+        request,
+        pk,
+        start=timezone.now().date(),
```

Ruff (0.8.2)

Do not perform function call in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)